fuel-relayer/README.md

```diff
@@ -25,105 +18,39 @@ Example of sliding window:
 ![Sliding Window](../docs/diagrams/fuel_v2_relayer_sliding_window.jpg)
```

Comment:
@Voxelot Should I try and make a new diagram as this one mentions staking etc?

Reply:
Yeah, we can also just remove the diagram if we think the current approach is simple enough to explain without it.